substrate/frame/identity/src/lib.rs

```diff
@@ -76,7 +76,7 @@ mod benchmarking;
 pub mod legacy;
 #[cfg(test)]
 mod tests;
-mod types;
+pub mod types;
```

Comment:
Is this necessary? We export specific types using `pub use` below (L93).

Reply:
This is left over from something old I was trying. Removing.